README.md

```diff
@@ -41,11 +41,14 @@ Install the CLI:
 cargo install --locked --path sim-cli/
 ```
 
-Run with Simulation File (see [setup instructions](#simulation-file-setup) for details):
- ```
-sim-cli sim.json
+Run a simulation
```

Comment:
Reading through this, the user doesn't yet know that they need to provide a simulation file. 

To run the simulation, create a simulation file `sim.json` in the working directory (see [setup instructions](#simulation-file-setup) for details) and run: 
```
sim-cli
```

Would also be nice to have some extra hints with `sim-cli -h` instructions like: 

Run `sim-cli -h` for details on  `--data-dir` and `--sim-file` options that allow specifying custom simulation file locations.